libs/sdk-core/src/breez_services.rs

```diff
@@ -2073,6 +2068,45 @@ impl BreezServicesBuilder {
             persister: persister.clone(),
         });
 
+        // mempool space is used to monitor the chain
+        let mempoolspace_urls = match self.config.mempoolspace_url.clone() {
+            None => {
+                let cached = persister.get_mempoolspace_base_urls()?;
+                match cached.len() {
+                    0 => {
+                        // If we have no cached values, or we cached an empty list, fetch new ones
+
+                        let fresh_urls = breez_server.fetch_mempoolspace_urls().await?;
+                        persister.set_mempoolspace_base_urls(fresh_urls.clone())?;
+                        fresh_urls
+                    }
+                    _ => {
+                        // If we already have cached values, return those
+
+                        // Start thread that refreshes cache in the background
```

Comment:
The only disadvantage I see to this is that these are not then available to the current instance, only when disconnecting/connecting. But I guess that list will not change too much

Reply:
Yes, that's the downside. The benefit is somewhat faster startup times, as we're not waiting for this call.